connector/MongoDBAtlasODBC.pq

```diff
@@ -5,7 +5,7 @@ section MongoDBAtlasODBC;
 // This should be set to false before release. Tracing is done through a call to 
 // Diagnostics.LogValue(). When EnableTraceOutput is set to false, the call becomes a 
 // no-op and simply returns the original value.
-EnableTraceOutput = true;
+EnableTraceOutput = false;

```

Comment:
Added a drive-by fix to disable tracing now that we will be submitting the connector for certification.

Reply:
ahah, I see, one less JIRA to deal with :)